### DISCUSSION TITLE
Production ready?

### DISCUSSION THREAD

**Discussion by pikitgb** [salvo-rs/salvo#127] [General]
I'm loving this project, do you recommend it for  production applications ?
Thanks a lot

**Comment by chrislearn** [comment]
I use it in production applications, and it works great.

**Reply by josejachuf** [reply]
Hello @chrislearn 
What main uses do you give to salvo?

**Reply by chrislearn** [reply]
Daily backend server development. I used nodejs and golang before, but all servers are using Rust now.

**Reply by josejachuf** [reply]
Thanks @chrislearn 

I think you use databases. Which one? 
With orm? Which one?
